homeassistant/components/evohome/climate.py

```diff
@@ -295,17 +284,26 @@ def __init__(self, evo_data, client, obj_ref):
         self._id = obj_ref.systemId
         self._name = '_{}'.format(obj_ref.location.name)
         self._icon = "mdi:thermostat"
-        self._type = EVO_PARENT
 
         self._config = evo_data['config'][GWS][0][TCS][0]
         self._status = evo_data['status']
         self._timers['statusUpdated'] = datetime.min
 
-        self._operation_list = TCS_OP_LIST
+        self._operation_list = list(HA_STATE_TO_TCS)
         self._supported_features = \
             SUPPORT_OPERATION_MODE | \
             SUPPORT_AWAY_MODE
 
+    @callback
+    def _refresh(self, packet):
+        if packet['signal'] == 'first_update':
+            self.schedule_update_ha_state(force_refresh=True)
```

Comment:
Use the async api in async context, ie `self.async_schedule_update_ha_state`.

Reply:
Sorry, a bad assumption on my part - entity uses `update()` and not `async_update()`.

Made the change you suggested.